packages/amplication-git-utils/src/utils/prepare-files-for-pull-request.ts

```diff
@@ -0,0 +1,82 @@
+import { join } from "path";
+import { Changes } from "octokit-plugin-create-pull-request/dist-types/types";
+import { AMPLICATION_IGNORED_FOLDER } from "../git/git.constants";
+import { AmplicationIgnoreManger } from "./amplication-ignore-manger";
+
+export async function prepareFilesForPullRequest(
+  owner,
+  repositoryName,
+  gitResourceMeta,
+  pullRequestModule
+): Promise<Required<Changes["files"]>> {
+  const amplicationIgnoreManger = new AmplicationIgnoreManger();
+  await amplicationIgnoreManger.init(async (fileName) => {
+    try {
+      const file = await this.getFile({
+        owner,
+        repositoryUrl: repositoryName,
+        path: fileName,
+        baseBranch: undefined, // take the default branch
+      });
+      const { content, htmlUrl, name } = file;
+      console.log(`Got ${name} file ${htmlUrl}`);
```

Comment:
would it be better to write to log instead of console

Reply:
again, this is not a new code